sky/engine/bindings/dart_ui.cc

```diff
@@ -50,6 +51,7 @@ void DartUI::InitForGlobal() {
     CanvasImage::RegisterNatives(g_natives);
     CanvasPath::RegisterNatives(g_natives);
     ColorFilter::RegisterNatives(g_natives);
+    DartJni::RegisterNatives(g_natives);
```

Comment:
#ifdef OS(ANDROID) ?

Also, please add a TODO about moving this into `dart:jni`.  It's ok to have it in `dart:ui` for the time being.